tools/testbench/file.c

```diff
@@ -409,6 +412,8 @@ static struct comp_dev *file_new(const struct comp_driver *drv,
 		(struct sof_ipc_comp_file *)comp;
 	struct file_comp_data *cd;
 
+	comp_cl_info(&comp_file, "file_new()");
```

Comment:
I remembered this after submitting. My common sense says this should be &comp_file_dai or &comp_file_host. However the code builds and testbench trace output shows file_new() when the components are instantiated by topology. 

I'm not familiar with the new trace functions so need to check this.

Reply:
I'll change this into debug_print() to avoid future build fail problems.

Reply:
This is fixed now as debug_print(). The comp_cl_info() macro did not build fail with &comp_file typo so need to be extra careful with those.

There were no other similar cases. I wanted to have this "trace" to see in future the pipeline walks also for only testbench related components for easier fail debugging.